app/src/main/java/org/fossasia/susi/ai/signup/SignUpActivity.kt

```diff
@@ -37,6 +41,7 @@ class SignUpActivity : AppCompatActivity(), ISignUpView {
     private lateinit var forgotPasswordProgressDialog: Dialog
     private lateinit var builder: AlertDialog.Builder
     private var checkDialog: Boolean = false
+    private val RECAPTCHA_KEY = "6LcKhbMUAAAAAGFbYZeNFzqol-7EjOHUK5MvEeOE"
```

Comment:
This shouldn't be hardcoded

Reply:
What else can I do to avoid this @iamareebjamal

Reply:
See where all configs are placed